code/game/gamemodes/changeling/modularchangling.dm

```diff
@@ -64,21 +64,21 @@ var/list/datum/power/changeling/powerinstances = list()
 
 /datum/power/changeling/deaf_sting
 	name = "Deaf Sting"
-	desc = "We silently sting a human, completely deafening them for a short time."
+	desc = "We sting a human, completely deafening them for a short time."
 	genomecost = 1
 	allowduringlesserform = 1
 	verbpath = /mob/proc/changeling_deaf_sting
 
 /datum/power/changeling/blind_sting
-	name = "Blind Sting"
-	desc = "We silently sting a human, completely blinding them for a short time."
+	name = "Blinding Spiders"
```

Comment:
Actually, better question: why is this called, "Blinding spiders" instead of "Blind Sting"? Even if the emote concerns spooders, just have it has a sting for clarity's sake, otherwise they'll think it actually spawns spiders that do things.